tests/tests.py

```diff
@@ -673,6 +673,40 @@ def test_remote_gs():
     run(dpath("test_remote_gs"))
 
 
+@pytest.mark.skip(reason="Need to choose how to provide billable project")
+@connected
+@not_ci
+def test_pr79(
```

Comment:
@johanneskoester is test_pr<number> an ok convention to use? If there aren't other test_pr* folders for GitLab we are probably good, and can assume that any test_pr folders are for GitHub. @jaicher another option would be to open an issue for this particular thing (if one isn't already) and reference it directly. If there isn't an issue, hats off to you for identifying an issue and going straight to a fix for it! :tophat:

Reply:
@vsoch I just created issue #96 and pushed a commit renaming the tests to reference this issue as suggested.

Reply:
Yes, saw it! I added a small comment about the naming convention (for GitHub vs Bitbucket) but otherwise we should be good. :)